cocos/2d/renderer/batcher-2d.ts

```diff
@@ -866,7 +867,10 @@ export class Batcher2D implements IBatcher {
     // TODO: Not a good way to do the job
     // Although it's a private method, it is invoked in text-processing.ts and texture-base.ts
     // by legacyCC.director.root.batcher2D._releaseDescriptorSetCache
-    private _releaseDescriptorSetCache (textureHash: number | Texture, sampler: Sampler | null = null): void {
+    /**
+     * @engineInternal
+     */
+    public _releaseDescriptorSetCache (textureHash: number | Texture | null, sampler: Sampler | null = null): void {
```

Comment:
1. It is better to modify the comment too, as it is not a private method now.
2. Why add type `null ` for textureHash?

Reply:
> 1
I will update it.

> 2
Because it will access the object returned by `getGFXTexture`
```ts
public getGFXTexture (): Texture | null {
```